tests/models.py

```diff
@@ -51,3 +51,14 @@ class ParentModel(models.Model):
 class RootNode(models.Model):
     id = models.IntegerField(primary_key=True)
     parent_field = models.ForeignKey(ParentModel, on_delete=models.CASCADE)
+
+
+try:
+    import dirtyfields
```

Comment:
What's this about? `dirtyfields` is a hard requirement, no?

Reply:
I was told that DDA _supports_ models that use `dirtyfields` but doesn't require it.  I don't know if that's true in practice.